tst/EnergyPlus/unit/AdvancedAFN.unit.cc

```diff
@@ -85,28 +85,28 @@ TEST_F(EnergyPlusFixture, AirflowNetwork_AdvancedTest_Test1)
     int CurveNum;
 
     AirflowNetworkNumOfOccuVentCtrls = 1;
-    state.dataAirflowNetworkBalanceManager->OccupantVentilationControl.allocate(AirflowNetworkNumOfOccuVentCtrls);
-    state.dataAirflowNetworkBalanceManager->OccupantVentilationControl(1).MinOpeningTime = 4;
-    state.dataAirflowNetworkBalanceManager->OccupantVentilationControl(1).MinClosingTime = 4;
-    state.dataAirflowNetworkBalanceManager->OccupantVentilationControl(1).MinTimeControlOnly = true;
+    state->dataAirflowNetworkBalanceManager->OccupantVentilationControl.allocate(AirflowNetworkNumOfOccuVentCtrls);
```

Comment:
You can expect lots and lots and lots of these changes if you are perusing this PR.  No functional change, just operating on a pointer rather than a direct instance of state.